library/network/src/modules/SuSEFirewall.rb

```diff
@@ -1204,6 +1204,15 @@ def IsServiceSupportedInZone(service, zone)
       return true
 
     end
+
+    # Function returns if firewall is protected from internal zone. For
+    # firewalld, we just return false.
```

Comment:
OK, it probably make sense to say GetProtectFromInternalZone == false,
but I was curious how this is used and found that the users of this module rely on fixed zone names: 
https://github.com/search?q=org%3Ayast+GetProtectFromInternalZone&type=Code

So that will need to be adapted too.